test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetIVsApiTests/NetCoreProjectTestCase.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using NuGet.StaFact;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Tests.Apex
+{
+    public class NetCoreProjectTestCase : SharedVisualStudioHostTestClass, IClassFixture<VisualStudioHostFixtureFactory>
```

Comment:
these cases are already part of other tests like install package? why do we need them separately as well?

Reply:
i just ported each and every test case that we had in powershell around netcore scenarios.